custom/icds_reports/utils/__init__.py

```diff
@@ -946,14 +968,29 @@ def create_aww_performance_excel_file(excel_data, data_type, month, state, distr
         headers.append("District")
     if aggregation_level < 3:
         headers.append("Block")
-    headers.extend(["Supervisor", "AWC", "AWW Name", "AWW Contact Number", "Home Visits Conducted",
-                    "Number of Days AWC was Open", "Weighing Efficiency", "Eligible for Incentive"])
-    columns = 'B C D E F G H I J'
-    if aggregation_level < 3:
-        columns += " K"
-    if aggregation_level < 2:
-        columns += " L"
-    columns = columns.split()
+
+    if beta:
+        headers.extend([
+            'Supervisor', 'AWC', 'AWW Name', 'AWW Contact Number',
+            'Home Visits Conducted', 'Weighing Efficiency', 'AWW Eligible for Incentive',
+            'Number of Days AWC was Open', 'AWH Eligible for Incentive'
+        ])
+        columns = 'B C D E F G H I J K'
```

Comment:
This could be much cleaner if it was just a list, instead of a string that gets added to and split later on